README.md

```diff
@@ -26,9 +26,40 @@ A large part Ozone Kh's roadmap is managed publicly to promote transparency and
 
 ## Release Notes
 
-### Version 1.7.0 (WIP)
+### Version 1.7.0
+
+- Updated project structure to be compatible with Ozone alpha.10 and above.
+- Removed OCL packages and retire Cancer concept.
+- Updated Common Reports to version 1.4.2
+- Fixed Route translation.
+- Included HPV virus test results in Cancer section in CSS form.
+- Updated project structure to be compatible with Ozone alpha.10 and above.
+- Adjusted aggregator artifact ID and remove relativePath.
+- Explicitly specify public repository in modules' pom.xml.
+- Disabled 'base' OpenMRS validation.
+- Validated on PR & self-rebuild upon dependency changes.
+- Compiled a dependency report for 'base' sub-module.
+- Renamed the Nexus server ID.
+- Set address and tags to Locations.
+- Copied OpenMRS configs for validation.
+- Fixed validator config path.
+- Removed empty curly brackets from Cervical Cancer Screening form.
+- Updated translation for 'History of Cervical Cancer Screening' in CCS form.
+- Updated question under Cancer section in CSS form.
```

Comment:
@kdaud, which ticket are you referring to for this note?

Reply:
KH-509, I've update the note.